Applications/OpenLIFUApp/slicer-application-properties.cmake

```diff
@@ -10,7 +10,7 @@ set(VERSION_MINOR
   1
   )
 set(VERSION_PATCH
-  1
+  2
```

Comment:
By the way, this is the version that the app will think it has and announce for itself. So after this version bump is merged I will push a v0.1.2 tag to match.